sources/packages/backend/apps/db-migrations/src/sql/Restrictions/Insert-institution-suspension-restriction.sql

```diff
@@ -0,0 +1,18 @@
+INSERT INTO
+  sims.restrictions (
+    restriction_type,
+    restriction_code,
+    description,
+    restriction_category,
+    action_type,
+    notification_type
+  )
+VALUES
+  (
+    'Institution' :: sims.restriction_types,
+    'SUS',
+    'Program suspended',
```

Comment:
Minor. I see that all the restriction descriptions end with period and we can follow the same here. 😊

Reply:
Added to keep similar to others, even though "periods" should be added to sentences, and some of them look like a "name" more than a sentence.